.ci/scripts/diff_dswx_files.py

```diff
@@ -0,0 +1,139 @@
+#   diff_dswx_files.py
+#
+#   For: OPERA project
+#
+#   11-03-2023 - Ray Bambery
+#       initial release
+#
+#   Description: Python program to perform Quality Assurance Test
+#       on a directory of NETcdf4 files - searches for files with
+#       a .tif file extension.
+#       This script calls a subprocess script,  dswx_comparison.py
+#       located in the dswx_s1_x.y.z directory shown as <DSWX-S1_DIR>
+#       in the wiki page 
+#       https://github.com/nasa/opera-sds-pge/wiki/DWSX_S1_Beta-Acceptance-Test-Instruuctions
+#       which ensures there are matching files in each input directory
+##
+#   Arguments:
+#       expected_output_dir output_dir dir
+#
+#   Outputs: A terminal stream of  results of the tests showing 
+#       PASS (in GREEN) or FAIL (in RED)
+#   
+#   Calls:  
+#       dswx_comparison.py
+#
+import os
+import sys
+import subprocess
+import argparse
+import glob
+#import netCDF4 as nc
+
+##################################################################
```

Comment:
I would prefer these kinds of dividers be removed, two lines of whitespace is sufficient for separating functions:

```
def func1():
    pass


def func2():
    pass
```